glide.lock

```diff
@@ -173,8 +173,12 @@ imports:
   version: a0d98a5f288019575c6d1f4bb1573fef2d1fcdc4
   subpackages:
   - simplelru
+- name: github.com/howeyc/gopass
```

Comment:
i dont know what this is. At least i didnt explicitely added it. I will see where it came from and remove it if possible.